src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -172,7 +173,9 @@ private static void setReportConfiguration(ReportConfiguration config, String ou
     }
 
     /**
-     * Set title of template file located at {@code filePath} to {@code pageTitle}
+     * Set title of template file located at {@code filePath} to {@code pageTitle}.
+     *
+     * @throws IOException if file writing fails.
```

Comment:
I think 'if writing to the file fails' would be better.

Reply:
Fixed.